front-office/Your-vouchers.md

```diff
@@ -0,0 +1,31 @@
+## Rules of the displayed vouchers
+
+A voucher is displayed if it follows specific restrictions:
+
+On the table below are listed all the different cases.
+
+| Cases | Code | Highlighted | Group restriction | User resctriction | Is Visible Voucher page ? |
+|-------|------|-------------|-------------------|-------------------|---------------------------|
+| 1     | X    | Yes         | X                 | X                 | Yes                       |
+| 2     | X    | X           | X                 | Yes               | Yes                       |
+| 3     | Yes  | No          | No                | No                | No                        |
+| 4     | No   | No          | Yes               | No                | Yes                       |
+| 5     | No   | No          | No                | No                | Yes                       |
+| 6     | Yes  | No          | Yes               | No                | No                        |
```

Comment:
What happens when the customer uses the voucher, is it still displayed on this page?

Reply:
One question left, sorry @MatShir! ⬆️

Reply:
From what I tested, if the "Total available for each user" is reched or if the "Total available" is 0, the voucher is still displayed but with 0 in the quantity column
I think we should specify this in the spec, see my suggestion at the line 21